device-registry/src/main/scala/org/genivi/sota/device_registry/db/DeviceRepository.scala

```diff
@@ -97,13 +98,17 @@ object DeviceRepository {
   }
 
   def updateLastSeen(uuid: Uuid)
```

Comment:
That also makes sense but also makes sense the repository knows the current time...